protected/humhub/modules/user/widgets/AccountTopMenu.php

```diff
@@ -95,6 +96,19 @@ public function init()
             'sortOrder' => 700,
         ]));
 
+        /* @var User $impersonator */
+        if ($impersonator = Yii::$app->user->getImpersonator()) {
+            $this->addEntry(new MenuLink([
+                'label' => Yii::t('base', 'Sign in back as {userName}', ['userName' => $impersonator->displayName]),
```

Comment:
Wording: `Stop impersonation`.

Reply:
Commit https://github.com/humhub/humhub/pull/5278/commits/7fb457ebdcb84934346639fcfdbd83983dc783fd

Reply:
@luke What do you think if we move all methods related to the impersonate action from the class `humhub\modules\user\components\User` into a separate behaviour like `humhub\modules\user\components\Impersonator`?

Reply:
@yurabakhtin Good idea, then it would be a little nicer separated. Can you please do this?

Reply:
@luke- Done in commit https://github.com/humhub/humhub/pull/5278/commits/87406048274102c9577a27a57d6e544a4efc3f0d.